js/renovation/ui/scroll_view/scrollable_simulated.tsx

```diff
@@ -171,11 +174,12 @@ export class ScrollableSimulated extends JSXComponent<ScrollableInternalPropsTyp
   }
 
   get cssClasses(): string {
-    const { direction, classes } = this.props;
+    const { direction, classes, disabled } = this.props;
 
     const classesMap = {
-      'dx-scrollable dx-scrollable-native dx-scrollable-renovated dx-scrollable-native-generic': true,
+      'dx-scrollable dx-scrollview dx-scrollable-simulated dx-scrollable-renovated': true,
```

Comment:
sx-scrollview should not be added always, it is for ScrollView component

Reply:
Fixed https://github.com/DevExpress/DevExtreme/pull/15636/commits/3371f7a4122918b13a658b8a49751be74f06fa71